DataFormats/Phase2L1Taus/src/L1HPSPFTau.cc

```diff
@@ -0,0 +1,95 @@
+#include "DataFormats/Phase2L1Taus/interface/L1HPSPFTau.h"
+
+// default constructor
+l1t::L1HPSPFTau::L1HPSPFTau() 
+  : tauType_(kUndefined)
```

Comment:
This doesn't look like the formatting that `clang-format`/`scram b code-format` would produce.
Can you run `scram b code-format` on your project area ?